tools.yaml

```diff
@@ -179,6 +179,38 @@ tools:
   - name: 'cutadapt'
     owner: 'lparsons'
     tool_panel_section_label: 'FASTA/FASTQ'
+    
+   - name: 'minimus2'
+     owner: 'edward-kirton'
+     tool_panel_section_label: 'Assembly'
+    
+   - name: 'fastp'
+     owner: 'iuc'
+     tool_panel_section_label: 'FASTA/FASTQ'
+     
+   - name: 'shasta'
+     owner: 'iuc'
+     tool_panel_section_label: 'Assembly'
+     
+  - name: 'trycycler'
+    owner: 'iuc'
+    tool_panel_section_label: 'Assembly'
+    
+  - name: 'orthofinder_onlygroups'
```

Comment:
I've replaced the suite with the individual tool. Suites are sometimes ill-behaved in galaxy.